core/types/rollup_l1_cost.go

```diff
@@ -45,10 +45,13 @@ type StateGetter interface {
 // Returns nil if there is no cost.
 type L1CostFunc func(blockNum uint64, blockTime uint64, dataGas RollupGasData, isDepositTx bool) *big.Int
 
+// TODO: are these slots updated / correct?
+// https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/L2/L1Block.sol#L39C28-L39C28
```

Comment:
These haven't changed, solidity packs storage slots automagically when it can, not all values are `uint256`

Reply:
I'm silly, that makes sense